src/scheduler.cpp

```diff
@@ -577,6 +567,11 @@ bool Scheduler::Worker::steal(Task& out) {
   return true;
 }
 
+Scheduler::Worker* Scheduler::Worker::getCurrent() {
```

Comment:
Is uninlining necessary here? `getCurrent()` is quite a hot method, so I think it would be better to keep this in the header.

Reply:
No it's not necessary.  Reverted back to inline in the latest revision.

Reply:
Great, thank you!